NoteFind/View Controllers/FeedViewController.m

```diff
@@ -7,6 +7,9 @@
 
 #import "FeedViewController.h"
 #import "NewNoteViewController.h"
+#import "Parse/Parse.h"
```

Comment:
nit: Framework style imports should be used for frameworks

`#import <Parse/Parse.h>`

Reply:
resolved!